### PR TITLE
Adds the --s3-only quick deploy feature.

### PR DESCRIPTION
**Pull request by mikegwhit**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = {
   ],
     rules: {
     // Remove any type warnings.
-    '@typescript-eslint/no-explicit-any':['error',{
+    '@typescript-eslint/no-explicit-any':['off',{
```
Comment (codeflyer) [inline .eslintrc.js:18]: A specific reason to set this value `off`?

**Review by codeflyer — COMMENTED** [review]
Just a not on the TS rules and it seem to miss the cloudfront invalidation action.

For the rest LGTM

File: .eslintrc.js (hunk 1 above)
Comment (mikegwhit) [reply .eslintrc.js:18]: This ended up being pretty difficult to work with.  In other words, as soon as we turn this on, we have to explicitly type everything.  I guess it's worth discussion, but my rule of thumb is that any time you have a meta-structure like test mocks or in this case typing where you exceed maybe a 2:3 ratio of meta-structure to code, you hit diminishing returns.  As it is with typing the CDK generated Cfn templates, I was having to type out these files.  Could they exist as a type within CDK?  Possibly.  I think it weighs down our development agility at a time when it is still getting discussed that Mira's purpose is not immediately clear to new developers.

Comment (codeflyer) [reply .eslintrc.js:18]: IDK what's the best approach. The `any error` was added with a specific task and if we remove it we'll have a regression.
Probably the best approach is to add some eslint-disable directive where the effort to add the types is useless and too expensive. In this way we can keep monitored what's not typed and avoid that parts that intstead requires a proper typing will be skipped.

Comment (mikegwhit) [reply .eslintrc.js]: Fixed

**Review by codeflyer — APPROVED** [review]

**Pull request merged**